packages/api/src/handlers/queues.ts

```diff
@@ -84,15 +85,31 @@ async function getAppQueues(
       const jobs = isActiveQueue
         ? await queue.getJobs(status, pagination.range.start, pagination.range.end)
         : [];
+      const formattedJobs = jobs.filter(Boolean).map((job) => formatJob(job, queue));
 
       const description = queue.getDescription() || undefined;
 
+      let filteredFormattedJobs:AppJob[] = [];
+      let filteredPagination:Pagination = pagination;
+      if (isFiltered) {
+        const filter = query.searchPrompt.toLocaleLowerCase();
+        const allJobs = isActiveQueue
+          ? await queue.getJobs(status)
```

Comment:
There is any other way to search the data in Redis or something like this?
I think that fetching all jobs in order to filter them out is not efficient.